themes/StarterTheme/templates/_partials/breadcrumb.tpl

```diff
@@ -0,0 +1,12 @@
+<nav>
+  <ol itemscope itemtype="http://schema.org/BreadcrumbList">
```

Comment:
this kind of notation could be like this using RDFa light:
<ol typeof="schema:BreadcrumbList">
instead of itemprop its property:
<span property="schema:name">